### ISSUE TITLE
InvalidOperationException: Sequence contains more than one element [HTTP 200 returned while handling GET Request]

### ISSUE THREAD

**Issue by dougwaldron** [gaepdit/airbranch-reports#41]
Raygun report https://app.raygun.com/crashreporting/29jeplo/errors/116149808569 has more details about the error

**Comment by dougwaldron** [comment]
A facility can have multiple ACCs from the same year:

```sql
select count(*)                    as [#],
       m.STRAIRSNUMBER             as airs,
       year(c.DATACCREPORTINGYEAR) as year
from dbo.SSCPITEMMASTER m
    inner join dbo.SSCPACCS c
    on m.STRTRACKINGNUMBER = c.STRTRACKINGNUMBER
where c.DATACCREPORTINGYEAR is not null
group by m.STRAIRSNUMBER, year(c.DATACCREPORTINGYEAR)
having count(*) > 1
order by year;
```

Current ACC method [assumes a unique result](https://github.com/gaepdit/airbranch-reports/blob/7c6eaa17faffec157d6fca2362c7d28ebe705d9b/src/Infrastructure/Compliance/ComplianceRepository.cs#L51).

**Comment by dougwaldron** [comment]
Raygun report https://app.raygun.com/crashreporting/29jeplo/errors/116230680068 has more details about the error

**Comment by dougwaldron** [comment]
IAIP fix: gaepdit/iaip@c229c0ab67e2755fe157564ba49da7c4b13a8cab
